test/integration/associations/belongs-to-many.test.js

```diff
@@ -206,7 +206,11 @@ describe(Support.getTestDialectTeaser('BelongsToMany'), function() {
       return self.sequelize.dropAllSchemas().then(function() {
         return self.sequelize.createSchema('acme');
       }).then(function() {
-        return self.sequelize.sync({force: true});
+        return AcmeUser.sync({force: true});
```

Comment:
Why the individual sync here? Sequelize should be able to figure out the right order